### PR TITLE
Added objects. Validate letters. Add distributions

### PR DESCRIPTION
**Pull request by peternijssen**
My enviroment is acting strange. Will have to check what's wrong, but perhaps meanwhile you can validate code. Thanks!

### CODE REVIEW

**Comment by peternijssen** [comment]
@iMicknl Any updates on code validation? I will run some final tests on my end.

**Comment by iMicknl** [comment]
Sorry, I totally missed this PR... Will have a look today / tomorrow! Thanks for your contribution.

**Comment by peternijssen** [comment]
No problem! I still have to look why it won't run locally. I believe it's a config error in my editor though. So I don't expect many more code changes.

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(name='postnl_api',
-      version='1.0.2',
+      version='2.0.0',
```
Comment (iMicknl) [inline setup.py]: Could you bump to 1.1.0? It doesn't look like there are any breaking changes.
Please also update the CHANGELOG.md.

**Comment by peternijssen** [comment]
Closing this one in favor of #11

**Comment by iMicknl** [comment]
@peternijssen I will have a look at all changes this week, so we can get it published and create a new Home Assistant PR.

**Pull request closed without merging**